drivers/aks/aks_driver.go

```diff
@@ -311,11 +331,58 @@ func (d *Driver) GetDriverCreateOptions(ctx context.Context) (*types.DriverFlags
 		Usage: "Specify the version of Kubernetes.",
 		Value: "1.11.5",
 	}
-	driverFlag.Options["tags"] = &types.Flag{
+	driverFlag.Options["enable-auto-scaling"] = &types.Flag{
+		Type:  types.BoolType,
+		Usage: "Should we enable auto-scaling.",
+		Default: &types.Default{
+			DefaultBool: false,
+		},
+	}
+	driverFlag.Options["min-count"] = &types.Flag{
+		Type:  types.IntType,
+		Usage: "Minimum number of nodes in the cluster (when auto-scaling is enabled).",
+		Default: &types.Default{
+			DefaultInt: 1,
+		},
+	}
+	driverFlag.Options["max-count"] = &types.Flag{
+		Type:  types.IntType,
+		Usage: "Maximum number of nodes in the cluster (when auto-scaling is enabled).",
+		Default: &types.Default{
+			DefaultInt: 3,
+		},
+	}
+	driverFlag.Options["agent-pool-type"] = &types.Flag{
+		Type:  types.StringType,
+		Usage: "Agent pool type (VirtualMachineScaleSets or AvailabilitySet).",
+		Value: "AvailabilitySet",
+	}
+	driverFlag.Options["availability-zones"] = &types.Flag{
 		Type:  types.StringSliceType,
+		Usage: "Availability zones to setup agent pools for.",
+		Default: &types.Default{
+			DefaultStringSlice: &types.StringSlice{Value: []string{"1"}}, //avoid nil value for init
+		},
+	}
+	driverFlag.Options["load-balancer-sku"] = &types.Flag{
+		Type:  types.StringType,
+		Usage: "Load balancer SKU (Basic or Standard).",
+		Value: "Basic",
```

Comment:
This should be `basic` I think. It is lower-case in model of azure sdk.

Reply:
Resolved :+1: